### PR TITLE
Помощь друга

### PR DESCRIPTION
**Pull request by Dzoter**
---
:mortar_board: [Помощь друга](https://up.htmlacademy.ru/javascript/27/user/1658553/tasks/16)

:boom: https://htmlacademy-javascript.github.io/1658553-kekstagram-simple-27/9/

### CODE REVIEW

File: js/project-src/send-edit-form/custom-slider.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
+const slideElement = document.querySelector('.effect-level__slider');
+const sliderField = document.querySelector('.img-upload__effect-level');
+const effectElementsCount = document.querySelector('.effects__list').children.length;
+
+const effectElementLevelInput = document.querySelector('.effect-level__value');
+
+const bigImg = document.querySelector('.img-upload__preview').children[0];
+
+const EFFECT_LIST = {
+  effect_chrome: 'chrome',
```
Comment (ademaro) [inline js/project-src/send-edit-form/custom-slider.js]: effectСhrome

File: js/project-src/send-edit-form/custom-slider.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  step: 1,
+  connetct: 'lower',
+});
+
+slideElement.noUiSlider.on('update', function (_,handle,unencoded) {
+  let checkedElement = document.querySelectorAll('input[name=effect]:checked')[0]
+
+  effectElementLevelInput.value = unencoded[handle];
+  if (checkedElement.value===EFFECT_LIST.effect_chrome){
+    bigImg.style.filter = `grayscale(${effectElementLevelInput.value})`
```
Comment (ademaro) [inline js/project-src/send-edit-form/custom-slider.js]: `bigImg.style.filter = EFFECTS[checkedElement.value]()`

File: js/project-src/send-edit-form/send-form.js [hunk 3]
```diff
@@ -16,67 +17,59 @@ const biggerButton = document.querySelector('.scale__control--bigger');
...
-
-smallerButton.addEventListener('click',changeSizeUploadedImgToLow)
-
-
 const validateType = function (file) {
   return file === 'image/jpeg' || file === 'image/png';
 };
 
+// eslint-disable-next-line no-console
+const closeImgEditorPopup = function () {
```
Comment (ademaro) [inline js/project-src/send-edit-form/send-form.js]: `function closeImgEditorPopup() {`

Comment (ademaro) [reply js/project-src/send-edit-form/send-form.js]: `const closeImgEditorPopup = () => {`

File: js/project-src/send-edit-form/custom-slider.js (hunk 1 above)
Comment (Dzoter) [reply js/project-src/send-edit-form/custom-slider.js]: done

File: js/project-src/send-edit-form/custom-slider.js (hunk 2 above)
Comment (Dzoter) [reply js/project-src/send-edit-form/custom-slider.js]: done

**Review by ademaro — APPROVED** [review]

**Pull request merged**